apps/website/blog/authors.yml

```diff
@@ -51,3 +51,9 @@ tumido:
   title: Maintainer of Janus Helm Charts & Plugins
   url: https://github.com/tumido
   image_url: https://github.com/tumido.png
+
+jayfray12:
+  name: Jason Froehlich
+  title: Contributor
```

Comment:
I wasn't sure what to put here.  I was going to put 'Red Hat Architect' but it seems as though nobody else used there official Red Hat title.  If there is something specific you'd like me to change it to let me know